Ромахина Галина

GalinaRoma opened the pull request:
[Посмотреть решение](https://rawgit.com/GalinaRoma/markup-task-3/master/index.html)

GalinaRoma committed:
Первое решение задачи

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bratva commented on index.css:
```diff
@@ -0,0 +1,196 @@
+    background: #000;
+}
+
+input[value='small']:checked ~ .page :nth-child(n)
```
очень странный селектор. Почему не просто `input[value='small']:checked ~ .page` ?

GalinaRoma replied:
Может быть проблема у меня, но Ваш вариант не работает.
Работает либо так, как я написала, либо ещё с *

bratva replied:
Дело в том, что данным селектором, ты переопределяешь стили каждого потомка внутри .page (это равносильно *). Данный селектор очень плох и его рекомендуется избегать, т.к. это может сказаться на производительности больших страниц.

Тебе нужно сделать иначе: Задать размер шрифта элементу .page, а дети пускай наследуют его и используют (%, em, ...). Подсмотри как это сделано у других ребят

bratva commented on index.html:
```diff
@@ -3,7 +3,135 @@
+    <main>
+        <header>
+            <h1>The Ural Times</h1>
+            <h6>Понедельник, 23 октября 2017г.</h6>
```
мало похоже на заголовок. Скорее это дата

bratva commented on index.html:
```diff
@@ -3,7 +3,135 @@
+                    а так&shy;же в со&shy;об&shy;ще&shy;стве
+                    Noize MC в VK.</p>
+            </section>
+            <section class="quote">
```
для цитат есть более семантичный тег

bratva commented on index.css:
```diff
@@ -0,0 +1,196 @@
+    text-transform: uppercase;
+    font: 12px Times New Roman, serif;
+    font-weight: 800;
+    font-variant: small-caps;
```
все это можно склеить в правило font

bratva commented on index.css:
```diff
@@ -0,0 +1,196 @@
+{
+    text-align: center;
+    text-transform: uppercase;
+    font: 30px Fenwick;
```
А если моя система не знает таких шрифтов?

bratva reviewed (commented):
Привет.

1. Не вижу вертикальной надписи
2. Не вижу подключение нестандартных шрифтов
3. Посмотри пжл. еще раз на значение тегов main/section/article - есть замечания по их использованию
🍅

GalinaRoma committed:
Добавила вертикальную надпись и нестандартный шрифт;
скеила в font;
поменяла тег у цитаты;
изменила селектор для звездочки;
изменила структуру документа.

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

GalinaRoma committed:
fixed

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты
